gen-certs.sh

```diff
@@ -18,7 +18,7 @@ certs() {
 }
 
 privatedir() {
-    echo "/etc/pki/private"
+    echo "$(dir)/private"
```

Comment:
Yes, this can be changed to a variable without much hassle (this and other functions).

This PR doesn't need to be part of the release, so I'll tidy it up afterwards.